src/layout/IHyprLayout.cpp

```diff
@@ -241,6 +241,9 @@ void IHyprLayout::onBeginDragWindow() {
     g_pHyprRenderer->damageWindow(DRAGGINGWINDOW);
 
     g_pKeybindManager->shadowKeybinds();
+
+    g_pCompositor->focusWindow(DRAGGINGWINDOW);
```

Comment:
don't think this is necessary it's already focused

Reply:
you can currently resize without focusing , the focus is only given in the end.
this way you can see which window you're resizing

Reply:
w8 wat you can? okay nvm then